src/commands/blobs/blobs.ts

```diff
@@ -53,7 +53,8 @@ export const createBlobsCommand = (program: BaseCommand) => {
       '-p, --prefix <prefix>',
       `A string for filtering down the entries; when specified, only the entries whose key starts with that prefix are returned`,
     )
-    .option('--json', `Output list contents as JSON`)
+    // The BaseCommand defines a `--json` option which is hidden from the help by default
+    .addHelpOption(new Option('--json', 'Output list contents as JSON'))
```

Comment:
Same as above here, but in this case it's intentional. Just removing this line works, but then `--json` no longer shows up in `ntl blobs:get --help`. This solution seems to cover all the bases.

Reply:
... and then same for all the command `--offline` cases

Reply:
What does "same as above here" refer to? Can you spell out what you mean?

Reply:
Oops, got it.